lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -586,7 +586,7 @@ def verify_flow_rules_deletion(self):
             dpath, {"table_id": self.SPGW_TABLE, "priority": 0}
```

Comment:
this`verify_flow_rules_deletion` function seems odd, I'm assuming we're verifying that all rules were removed? Should we remove the priority match here completely then?

Or we could potentially query rules of different priorities (assert that no UE rules are left)
@ulaskozat

Reply:
Lets merge this PR to unblock CI though

Reply:
let's merge it and can discuss after unblocked for the best way forward.

Reply:
but yes, why do we check against priority 0? @pruthvihebbani

Reply:
@koolzz on table zero no rules but the default rule(s) should remain. removing "priority":0 in the match criteria should fix the logic.

Reply:
sure nuked the priority match lets see

Reply:
I took this piece of code from test_attach_detach_rar_tcp_data.py TC assuming that flows with priority 0 are the default rules which are there even when the TCs are not running

Reply:
@pruthvihebbani raised and merged PR #7013.